includes/specials/SpecialUserlogout.php

```diff
@@ -33,15 +33,18 @@ function __construct() {
 	}
 
 	function execute( $par ) {
-		global $wgUser;		/* wikia change */
+		global $wgUser;        /* wikia change */
 
 		/**
 		 * Some satellite ISPs use broken precaching schemes that log people out straight after
 		 * they're logged in (bug 17790). Luckily, there's a way to detect such requests.
 		 */
-		if ( isset( $_SERVER['REQUEST_URI'] ) && strpos( $_SERVER['REQUEST_URI'], '&amp;' ) !== false ) {
+		if ( isset( $_SERVER['REQUEST_URI'] ) &&
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "REQUEST_URI" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)